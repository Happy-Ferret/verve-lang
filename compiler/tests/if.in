(if (equals 1 1)
    (print 1)
    (print 2))
