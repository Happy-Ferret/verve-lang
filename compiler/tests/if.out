false
8
8
done
